usefulUtilities/bouncerZone/bouncerZone.js

```diff
@@ -202,35 +191,9 @@
             }
             return false;
         }
-
-        // Determines whether or not my username is on the whitelist
-        // populated at the top of this script. 
-        function isOnHardcodedWhitelist() {
-            if (APPROVED_USERNAMES.length === 0) {
-                return false;
-            }
-
-            var currentUsername = AccountServices.username.toLowerCase();
-            var lowerCaseHardcodedUsernames = APPROVED_USERNAMES.map(function(value) {
-                return value.toLowerCase();
-            });
-
-            if (lowerCaseHardcodedUsernames.indexOf(currentUsername) >= 0) {
-                if (DEBUG) {
-                    print("Username is on hardcoded whitelist");
-                }
-                return true;
-            } else {
-                return false;
-            }
-        }
             
-        if (isOnUserDataUsernameWhitelist() || isOnHardcodedWhitelist()) {
-            // Do nothing; allow the avatar to stay in the Bouncer Zone
-        } else if (ticketMarketplaceID !== "") {
-            // If ticketMarketplaceID is defined, start by searching my avatar for the ticket wearable
-            searchForTicketWearable();
-        } else {
+        if (!isOnUserDataUsernameWhitelist() && !(allowAdmins && Entities.canAdjustLocks()) && 
```

Comment:
Wait, shouldn't all of these be `OR`s, not `AND`s? Why are these separated by `&&`?

Reply:
no, I changed it to check if any conditions exist to NOT bounce the user

Reply:
OK, the logic works